wordpress-mobile/WordPress-iOS
# Blogging Prompts Card: Fix avatar train color

Refs https://github.com/wordpress-mobile/WordPress-iOS/pull/18440#issuecomment-1109910763

This updates the color and border for the avatar train. Here are some screenshots:

• | Before | After
--|--|--
Feature Introduction, light — should be unchanged | ![fi-light-before](https://user-images.githubusercontent.com/1299411/165778920-f7772bdb-e3ca-4725-87f7-d07cfe8230b1.png) | ![fi-light-after2](https://user-images.githubusercontent.com/1299411/165780404-d51a2f42-fca4-4db0-95d2-15e6a1c03b6b.png)
Feature Introduction, dark | ![fi-dark-before](https://user-images.githubusercontent.com/1299411/165778901-b07cd033-2f19-498a-84f6-2ecf58af9821.png) | ![fi-dark-after](https://user-images.githubusercontent.com/1299411/165778936-e72ba777-ae58-4266-8026-e02e25f962f7.png)
Dashboard card, light  — should be unchanged| ![card-light-before](https://user-images.githubusercontent.com/1299411/165778911-7c9aff90-1d5b-4af0-926b-cccf82d74fa1.png) | ![card-light-after2](https://user-images.githubusercontent.com/1299411/165780429-0c6ccadc-6718-4106-8df7-409274fbddfa.png)
Dashboard card, dark | ![card-dark-before](https://user-images.githubusercontent.com/1299411/165778890-f9140bd1-7b91-4c49-82ff-89475e2ec0c8.png) | ![card-dark-after](https://user-images.githubusercontent.com/1299411/165778931-b3334371-b674-41a2-a1f6-e636d6b8fcc1.png)

## To test

- Enable the Blogging Prompts feature flag.
- Verify that in the Feature Introduction sheet, the avatar train's border now blends with the background color.
- Verify that the default avatar colors are displayed correctly without its neighbors [peeking](https://cldup.com/a0PuFmAAdB.png) (colors from the left image views are seeping into the right image views).
- Change the user interface style.
- Verify that the colors are displayed correctly for the new user interface style in the dashboard card and the Feature Introduction.

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is unreleased.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is unreleased.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is unreleased.

PR submission checklist:
